components/options_resolver.rst

```diff
@@ -686,6 +686,16 @@ default value::
         ],
     ]);
 
+.. deprecated:: 7.3
+
+    Defining nested option via :method:`Symfony\\Component\\OptionsResolver\\OptionsResolver::setDefault`
+    was deprecated since Symfony 7.3, use the :method:`Symfony\\Component\\OptionsResolver\\OptionsResolver::setOptions`
+    instead. Allowing to define default values for prototyped options.
```

Comment:
I'm not sure about this deprecation block. Is it needed in this case, or would the replacement + versionadded note be enough?

Reply:
Yes, sometimes it's difficult to decide if we should add such a block. At the end, I left it so anyone searching for "setDefault" can still find this page. Thanks!